python/paddle/fluid/tests/unittests/ir/test_convert_to_mixed_precision.py

```diff
@@ -30,71 +32,70 @@
-    def test_convert_to_fp16(self):
+    def setUp(self):
+        self.temp_dir = tempfile.TemporaryDirectory()
         model = resnet50(True)
         net = to_static(
             model, input_spec=[InputSpec(shape=[None, 3, 224, 224], name='x')]
         )
-        paddle.jit.save(net, 'resnet50/inference')
+        paddle.jit.save(
+            net, os.path.join(self.temp_dir.name, 'resnet50/inference')
+        )
+
+    def tearDown(self):
+        self.temp_dir.cleanup()
+
+    def convert_to_fp16(self):
         convert_to_mixed_precision(
-            'resnet50/inference.pdmodel',
-            'resnet50/inference.pdiparams',
-            'mixed/inference.pdmodel',
-            'mixed/inference.pdiparams',
+            os.path.join(self.temp_dir.name, 'resnet50/inference.pdmodel'),
+            os.path.join(self.temp_dir.name, 'resnet50/inference.pdiparams'),
+            os.path.join(self.temp_dir.name, 'mixed/inference.pdmodel'),
+            os.path.join(self.temp_dir.name, 'mixed/inference.pdiparams'),
             PrecisionType.Half,
             PlaceType.GPU,
             True,
         )
 
-    def test_convert_to_fp16_with_fp16_input(self):
-        model = resnet50(True)
-        net = to_static(
-            model, input_spec=[InputSpec(shape=[None, 3, 224, 224], name='x')]
-        )
-        paddle.jit.save(net, 'resnet50/inference')
+    def convert_to_fp16_with_fp16_input(self):
         convert_to_mixed_precision(
-            'resnet50/inference.pdmodel',
-            'resnet50/inference.pdiparams',
-            'mixed1/inference.pdmodel',
-            'mixed1/inference.pdiparams',
+            os.path.join(self.temp_dir.name, 'resnet50/inference.pdmodel'),
+            os.path.join(self.temp_dir.name, 'resnet50/inference.pdiparams'),
+            os.path.join(self.temp_dir.name, 'mixed1/inference.pdmodel'),
+            os.path.join(self.temp_dir.name, 'mixed1/inference.pdiparams'),
             PrecisionType.Half,
             PlaceType.GPU,
             False,
         )
 
-    def test_convert_to_fp16_with_blacklist(self):
-        model = resnet50(True)
-        net = to_static(
-            model, input_spec=[InputSpec(shape=[None, 3, 224, 224], name='x')]
-        )
-        paddle.jit.save(net, 'resnet50/inference')
+    def convert_to_fp16_with_blacklist(self):
         convert_to_mixed_precision(
-            'resnet50/inference.pdmodel',
-            'resnet50/inference.pdiparams',
-            'mixed2/inference.pdmodel',
-            'mixed2/inference.pdiparams',
+            os.path.join(self.temp_dir.name, 'resnet50/inference.pdmodel'),
+            os.path.join(self.temp_dir.name, 'resnet50/inference.pdiparams'),
+            os.path.join(self.temp_dir.name, 'mixed2/inference.pdmodel'),
+            os.path.join(self.temp_dir.name, 'mixed2/inference.pdiparams'),
             PrecisionType.Half,
             PlaceType.GPU,
             False,
             set('conv2d'),
```

Comment:
Is this expected? I bet you want `set(['conv2d'])`

```
Python 3.8.10 (default, Jun 22 2022, 20:18:18)
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> set('conv2d')
{'n', 'v', 'd', 'o', '2', 'c'}
>>> set(['conv2d'])
{'conv2d'}
```

Reply:
Thanks very much!